makefile

```
@@ -0,0 +1,10 @@
+dev_dependencies:
+	pip install -r requirements-dev.txt
+	pip install -e .
+
+test_dependencies:
+	pip install -r requirements.txt
+	pip install -e .
```

I think I'd suggest something like this:
```suggestion
dev_deps:
	pip install -r requirements-dev.txt

deps:
	pip install -r requirements.txt

build: deps
	pip install -e .

build_dev: dev_deps
	pip install -e .
```